syntax = "proto3";

package AcFunDanmu.Im.Message;

import "Message.proto";

message PullOldResponse {
	repeated Message messages = 1;
	int64 serverTime = 2;
}
